api/src/main/java/org/pmiops/workbench/cdr/dao/CBCriteriaDao.java

```diff
@@ -233,9 +233,9 @@
 
   @Query(
       value =
-          "select * from cb_criteria where domain_id = 'PERSON' and type in ('GENDER', 'RACE', 'ETHNICITY') and parent_id != 0 order by name asc",
+          "select * from cb_criteria where domain_id = 'PERSON' and type in ('GENDER', 'RACE', 'ETHNICITY', 'SEX') and parent_id != 0 order by name asc",
       nativeQuery = true)
-  List<DbCriteria> findGenderRaceEthnicity();
+  List<DbCriteria> findGenderRaceEthnicitySexAtBirth();
```

Comment:
Do you think the name findAllDemographics/findDempgraphics will be better?

Reply:
Done.